### DISCUSSION TITLE
why “Locations with the following IDs could not be found in the locations sheet”, when origin, dest, and count are marked in the csv file?

### DISCUSSION THREAD

**Discussion by meng-lingxin-199** [FlowmapBlue/FlowmapBlue#68] [Q&A] [answered]
Hi, ilyabo! Thank you so much for this great dynamic visualization.
I'm writing a paper on separation of work and residence, hoping to visualize the OD data through this excellent tool. But I have a problem, that is, whenever I try to run it, I get the following prompt (image 1). 
But I did enter the data in csv format as required, as shown below (image 2). Meanwhile, I will also attach my csv file.
Could you please help me check what the problem is? 
Besides, I hope you don't mind if my English expression doesn't sound polite or makes you feel offended. It's all because of my poor English.
Thanks in advance!
![image1](https://user-images.githubusercontent.com/112576846/187741985-f1e4a466-ff9a-4392-8b5e-fb9d87316f18.png)
![image2](https://user-images.githubusercontent.com/112576846/187741992-f4f50a47-b2d2-40e3-b4b7-edaece83928f.png)
[行政区-流csv-blue.csv](https://github.com/FlowmapBlue/FlowmapBlue/files/9463683/-.csv-blue.csv)

**Comment by GianfraGovUK** [comment]
Hello,
why don't you share also your locations CSV?
The flows CSV looks fine but from your image locations CSV has an additional column with names.
It should be just id, lat, lon. It seems to me you have name, id, lat, lon.
Maybe if you remove the first column of the locations CSV it will work. Otherwise you can replace the id used in the flows CSV with the names and remove the second column in the Locations CSV.

Good luck

Gianfranco

**Comment by ilyabo** [comment, marked as answer]
Exactly, the issue is that the id and name columns are confused. If you just change `id,name` to `name,id` in the locations CSV it should work.
